ios/SimpleDemo/Podfile

```diff
@@ -0,0 +1,6 @@
+platform :ios, '8.0'
+
+target ‘SimpleDemo' do
```

Comment:
I think this is the reason for this "[!] Your Podfile has had smart quotes sanitised. To avoid issues in the future, you should not use TextEdit for editing it. If you are not using TextEdit, you should turn off smart quotes in your editor of choice."